### PR TITLE
2022-05-08-Terra-Classic

### PR DESCRIPTION
**Pull request by JediFaust**

### CODE REVIEW

**Comment by JediFaust** [comment]
Added detailed timeline with sources

File: content/attacks/posts/2022-05-08-Terra_Classic.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- **May 27, 2023:** Blockchain data analytics firm Nansen releases a report on the UST's collapse, debunking the theory of a single attacker causing the depeg. [Source](https://www.coindesk.com/layer2/2022/05/27/nansen-research-debunks-single-attacker-myth-in-terras-collapse/)
+- **March 23, 2023:** Do Kwon, the CEO of Terraform Labs, [was arrested in Montenegro](https://edition.cnn.com/2023/03/23/tech/south-korea-do-kwon-terra-fraud-arrest-hnk-intl/index.html).
+- **May 28, 2023:** [Terra 2.0 launches](https://www.coindesk.com/tech/2022/05/27/new-terra-blockchain-expected-to-launch-on-saturday-luna-airdrop-to-follow/), with a LUNA airdrop planned to follow.
+
+
+## Security Failure Causes
+
+The crisis was the result of a combination of factors: poor decision-making by the Luna Foundation Guard (the decision to withdraw 250 million $USTC from Curve3pool), significant withdrawals by whales from the anchor protocol, an imbalance between $USTC and other stablecoins in the pool, a firesale of $USTC resulting in uncontrollable minting of $LUNC, and network overload. The systemic failure to predict and manage these risks led to the implosion of the Terra Classic Network.
```
Comment (evgenydmitriev) [inline content/attacks/posts/2022-05-08-Terra_Classic.md:44]: Please use the bullet point structure found in other published articles.

**Review by evgenydmitriev — CHANGES_REQUESTED** [review]

**Comment by evgenydmitriev** [comment]
Avoid any special characters apart from `-` in your file names.

**Comment by JediFaust** [comment]
@evgenydmitriev fixed all issues

**Review by jhirschkorn — APPROVED** [review]

**Review by jhirschkorn — APPROVED** [review]

**Review by evgenydmitriev — APPROVED** [review]

**Pull request merged**